service/controller/v11/resource/deployment/master_deployment.go

```diff
@@ -89,21 +89,20 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 				Replicas: &replicas,
 				Template: apiv1.PodTemplateSpec{
 					ObjectMeta: apismetav1.ObjectMeta{
-						GenerateName: key.MasterID,
-						Finalizers: []string{
-							key.DrainingNodesFinalizer,
```

Comment:
Note we do not need the additional finalizer. We only rely on the automatically managed one via operatorkit. This is super nice and makes it more simple for the pod resource.